nestkernel/connector_base.h

```diff
@@ -451,7 +450,7 @@ class Connector : public ConnectorBase
   void
   reserve( const size_t count )
   {
-    C_.reserve( count );
+    //    C_.reserve( count );
```

Comment:
maybe add a comment why this is commented out, or remove function entirely if not used?

Reply:
I removed the function entirely, as one can't (and shouldn't) reserve memory for the new container.